addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js

```diff
@@ -71,7 +71,7 @@ patch(Wysiwyg.prototype, 'wysiwyg_iframe.js', {
             this.$root = this.$editable;
         }
         this.$iframe = $('<iframe class="wysiwyg_iframe o_iframe">').css({
-            'min-height': '55vh',
+            'min-height': '66vh',
```

Comment:
Note that this affects all wysiwyg iframes so I think it should be mentioned in the commit message.

Not sure if it's still necessary since since we made it full-size any way (?)

Reply:
I think it's not necessary anymore so i just reverted the change. But I think it's better to keep this for safety ?

Reply:
> I think it's better to keep this for safety

You mean you'd like to keep the added height? If so I would make sure to only apply it for the mass mailing override of the field, otherwise I'd check with rd-framework-editor to make sure.